internal/receiver/smartagentreceiver/converter/event.go

```diff
@@ -58,6 +58,8 @@ func sfxEventToPDataLogs(event *event.Event, logger *zap.Logger) pdata.Logs {
 		attrs.InsertInt(SFxEventCategoryKey, int64(event.Category))
 	}
 
+	attrs.InsertString(SFxEventType, event.EventType)
```

Comment:
I think it will be useful to check `if event.EventType != "" {` to avoid creating an empty attribute if the EventType is not defined.

Reply:
Nit: also maybe change line 46 to `attrsCapacity := 2 + len(event.Dimensions)` to reserve one additional space for the new attribute (which likely is going to be present in the majority of cases).